core-library/src/main/java/org/silverpeas/core/contribution/content/form/displayers/WysiwygFCKFieldDisplayer.java

```diff
@@ -153,20 +153,25 @@ public void displayScripts(PrintWriter out, FieldTemplate template, PagesContext
   @Override
   public void display(PrintWriter out, TextField field, FieldTemplate template,
       PagesContext pageContext) throws FormException {
-    String code = "";
     String fieldName = template.getFieldName();
     Map<String, String> parameters = template.getParameters(pageContext.getLanguage());
-    if (!field.getTypeName().equals(TextField.TYPE)) {
-
-    }
-    if (!field.isNull()) {
-      code = field.getValue(pageContext.getLanguage());
-    }
 
     String contentLanguage = I18NHelper.checkLanguage(pageContext.getContentLanguage());
 
-    code = getContent(pageContext.getComponentId(), pageContext.getObjectId(), template.
-        getFieldName(), code, contentLanguage);
+    String code = getContent(pageContext.getComponentId(), pageContext.getObjectId(), fieldName,
+        contentLanguage);
+
+    if (code == null) {
+      // get content in other languages
+      List<String> languages = new ArrayList<String>(I18NHelper.getAllSupportedLanguages());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS2293)